README.md

```diff
@@ -43,3 +43,24 @@
 4. Install `sudo apt -y install iptables-persistent` and **select "Yes", save rules!**.
 5. Replace tables `sudo cp -r RaspberryWallet/etc/iptables /etc/`
 6. **BE WARNED** this can lock you out forever till reflash (or try via UART console if something goes wrong). *Reboot*.
+
+## Install Manager (replace with Release installer)
+1. Download latest Manager.jar release from github 
+```bash
+cd /opt/wallet && \
+curl -s https://api.github.com/repos/RaspberryWallet/Backend/releases/latest \
+| grep "browser_download_url.*jar" \
+| cut -d : -f 2,3 \
+| tr -d \" \
+| wget -qi -
+```
```

Comment:
change this spaghetti to
`sudo apt -y install jq`
`curl -s https://api.github.com/repos/RaspberryWallet/Backend/releases/latest | jq ".assets[0].browser_download_url"`

Reply:
is it worth additional package ?

Reply:
dunno, you guys like clean code so I suggested ;d